test-runner-poc/test/tools/utils.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const Logger = require('../../lib/core').Logger;
+const deprecateOptions = require('../../lib/utils').deprecateOptions;
+const chai = require('chai');
+const expect = chai.expect;
+const sinonChai = require('sinon-chai');
+chai.use(sinonChai);
+
+function makeTestFunction(config) {
+  const fn = options => {
+    if (options) options = null;
+  };
+  return deprecateOptions(config, fn);
+}
+
+function ensureCalledWith(stub, args) {
+  args.forEach(m => expect(stub).to.have.been.calledWith(m));
+}
+
+// creation of class with a logger
+function ClassWithLogger() {
```

Comment:
What are these Logger classes used for?

Reply:
In `deprecate_warning_tests.js` it `requires` the `utils` folder and also calls `utils.ClassWithLogger`, `utils.ClassWithoutLogger` and `utils.ClassWithUndefinedLogger`